spk/tcl/src/installer.sh

```diff
@@ -16,11 +19,10 @@ preinst ()
 postinst ()
 {
 	# Create symlinks
-	ln -s ${SYNOPKG_PKGDEST}/bin/tclsh8.6 /usr/local/bin/tclsh
-	ln -s ${SYNOPKG_PKGDEST}/lib/libtcl8.6.so /usr/local/lib/libtcl.so
-	ln -s ${SYNOPKG_PKGDEST}/lib/libtcl8.6.so /usr/local/lib/libtcl8.6.so
-	ln -s ${SYNOPKG_PKGDEST}/lib/tcl8.6/ /usr/local/lib/tcl8.6
-	export LD_LIBRARY_PATH=/usr/local/lib/
+	ln -s ${PACKAGE_DIR}/bin/tclsh8.6 /usr/local/bin/tclsh >> ${LOG_FILE} 2>&1
```

Comment:
Creating this symlink is fine, but the next ones, not so much ;) Please remove them, there's no need for them.

Also, you should verify that `/usr/local/bin` actually exists/is created if not. Check other packages for syntax.